clab/test_data/topo8-bind-nodedir.yml

```diff
@@ -0,0 +1,11 @@
+name: topo8
```

Comment:
this file seems not being used in any way?

Reply:
You're right, my first intention was to test this also via the TestBindsInit() method.
Changed my mind along the path. Will remove it!